### ISSUE TITLE
Unfound tags to be added in the style tag from the header

### ISSUE THREAD

**Issue by adyz** [jonkemp/gulp-inline-css#19]
Hi Guys, 

I'm planning to use this module to create an email and it works great, but with one problem.

If I have a tag that is not found in the HTML, eg: #outlook a {padding:0;} it's not added the the header style tag. It's just adding media query tags. 

In there something I'm going wrong or this is the best I can get from gulp-inline-css?

Thank you, 
Adrian

**Comment by jonkemp** [comment]
No, that's how it's designed to work. You could use `options.removeStyleTags: false` to preserve the style tag in the document. Maybe that would do what you are wanting? I think that is your only option.
